README.md

```diff
@@ -1,29 +1,48 @@
 # Knot.x Starter Kit
-Knot.x Starter Kit is a template project that you can use when creating some Knot.x extensions.  It 
-allows you to easily integrate all custom modules with [Knot.x Stack](https://github.com/Knotx/knotx-stack) and build a custom Docker image.
+Knot.x Starter Kit is a template project that simplifies Knot.x project setup. It allows you to customize
+[Knot.x Stack](https://github.com/Knotx/knotx-stack) and deliver it as a Docker image.
 
 ## How to start
-To start a new Knot.x project with extensions simply download this repository as a ZIP and unzip it.
+To start a new Knot.x project with extensions simply [download this repository as a ZIP](https://github.com/Knotx/knotx-starter-kit/archive/master.zip) and unzip it.
 
 You may also use the following command:
 ```
 git clone --depth=1 git@github.com:Knotx/knotx-starter-kit.git knotx-example && rm -rf !$/.git
 ```
 to clone the repository contents to the `knotx-example` directory and start without git history.
 
+## How to build Docker image?
+Simply run the command
+```
+./gradlew build
+```
+that copies
+- custom modules with transitive dependencies
+- configuration files
+- Dockerfile
+
+to `build` directory and build Docker image based on Dockerfile. 
+
+Then it starts container with the new image and validate functional tests.
+
+## How can I run my Docker image?
+```
+docker run -p8092:8092 knotx/knotx-starter-kit
```

Comment:
Short info about how to customize `knotx/knotx-starter-kit` image name would be good here.

Reply:
Sure, https://github.com/Knotx/knotx-starter-kit/issues/6. We need to rethink where some settings should be placed, like port number etc.